### ISSUE TITLE
Question : about the protocol

### ISSUE THREAD

**Issue by houssemko** [Vishram1123/gjs-osk#143]
What protocol do you use to make the osk work in chromium based browser on wayland session ? I wanna use the default osk that Gnome uses in accessibility section but it doesn't work in chromium .
Your extension works , is there a way to use your extension protocol but with the default OSK ?

**Comment by Vishram1123** [comment]
I have no idea why my keyboard works in Chromium but not the default OSK - I use the exact same Clutter.VirtualInputDevice that the regular keyboard uses.  Additionally, modifying the default OSK is gonna be pretty difficult - why don't you just use GJS-OSK? It implements all the features of the default keyboard, and more

**Comment by Vishram1123** [comment]
With the following keyboard layout, you can get this:

<img width="1328" height="358" alt="Image" src="https://github.com/user-attachments/assets/a61f644e-18f0-482d-b733-caa462afc911" />

<details>
<summary>Layout JSON (copy and add it in the "Custom Layouts" tab, then select it under the "Layouts" dropdown)</summary>
<pre>[[{"key":"TLDE","width":2},{"key":"AE01"},{"key":"AE02"},{"key":"AE03"},{"key":"AE04"},{"key":"AE05"},{"key":"AE06"},{"key":"AE07"},{"key":"AE08"},{"key":"AE09"},{"key":"AE10"},{"key":"AE11"},{"key":"AE12"},{"key":"BKSL","width":2}],[{"key":"TAB","width":1.5},{"key":"AD01"},{"key":"AD02"},{"key":"AD03"},{"key":"AD04"},{"key":"AD05"},{"key":"AD06"},{"key":"AD07"},{"key":"AD08"},{"key":"AD09"},{"key":"AD10"},{"key":"AD11"},{"key":"AD12"},{"key":"BKSP","width":2.5}],[{"key":"CAPS","width":2},{"key":"AC01"},{"key":"AC02"},{"key":"AC03"},{"key":"AC04"},{"key":"AC05"},{"key":"AC06"},{"key":"AC07"},{"key":"AC08"},{"key":"AC09"},{"key":"AC10"},{"key":"AC11"},{"key":"RTRN","width":3}],[{"key":"LFSH","width":2},{"key":"LSGT"},{"key":"AB01"},{"key":"AB02"},{"key":"AB03"},{"key":"AB04"},{"key":"AB05"},{"key":"AB06"},{"key":"AB07"},{"key":"AB08"},{"key":"AB09"},{"key":"AB10"},{"key":"UP"},{"key":"RTSH","width":2}],[{"key":"LCTL"},{"key":"LWIN"},{"key":"LALT"},{"key":"SPCE","width":9},{"key":"LEFT"},{"key":"DOWN"},{"key":"RGHT"},{"width":1,"height":1,"key":"PRSC"}],{"split":false,"settings":true,"close":true}]
</pre>

**Comment by Vishram1123** [comment]
(Closing this for now, mention me if you need anything else)

**Comment by houssemko** [comment]
Thank you so much
